.github/workflows/build.yaml

```diff
@@ -74,7 +74,20 @@ jobs:
 
     # Audit all currently installed packages for security vulnerabilities.
     - name: Audit installed packages
-      run: make audit
+      id: audit-packages
+      continue-on-error: true
+      run: make --silent audit > vulnerabilities.txt
+
+    # Upload the vulnerabilities file output.
+    - name: Upload Artifact
+      id: upload-audit-artifact
```

Comment:
This `id` isn’t referenced so, for simplicity’s sake, we can remove it. Same with the `id: comment-pr` below.

Reply:
Removed!